src/test/java/teammates/test/cases/ui/pagedata/FeedbackSubmissionEditPageDataTest.java

```diff
@@ -0,0 +1,142 @@
+package teammates.test.cases.ui.pagedata;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertFalse;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.AccountAttributes;
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.FeedbackResponseAttributes;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.datatransfer.FeedbackSessionQuestionsBundle;
+import teammates.common.datatransfer.StudentAttributes;
+import teammates.common.util.Const;
+import teammates.test.cases.BaseTestCase;
+import teammates.ui.controller.FeedbackSubmissionEditPageData;
+import teammates.ui.template.StudentFeedbackSubmissionEditQuestionsWithResponses;
+
+public class FeedbackSubmissionEditPageDataTest extends BaseTestCase {
+    private static DataBundle dataBundle = getTypicalDataBundle();
+    
+    @BeforeClass
+    public static void classSetUp() throws Exception {
+        printTestClassHeader();
+    }
+    
+    @Test
+    public void testAll() {
+        ______TS("test typical case");
+        AccountAttributes studentAccount = dataBundle.accounts.get("student1InCourse1");
+        StudentAttributes student = dataBundle.students.get("student1InCourse1");
+        FeedbackSubmissionEditPageData pageData = new FeedbackSubmissionEditPageData(studentAccount, student);
+        
+        FeedbackSessionAttributes feedbackSession = dataBundle.feedbackSessions.get("session1InCourse1");
+        FeedbackQuestionAttributes question = dataBundle.feedbackQuestions.get("qn1InSession1InCourse1");
+        List<FeedbackResponseAttributes> responses = new ArrayList<FeedbackResponseAttributes>();
```

Comment:
Would it be possible to just generally tidy this up with more methods (e.g. `createData`, `test___`)?
Storing data in `private` class-scoped variables is fine here since it'll be collected up after each test is done.
There seems to be a bit of code duplication too, so it'll help reduce that.